src/QMCTools/CMakeLists.txt

```diff
@@ -53,11 +53,10 @@ if(USE_OBJECT_TARGET)
   target_link_libraries(qmcfinitesize qmcparticle qmcutil)
 endif()
 
-file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/gpaw4qmcpack.py ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/gpaw4qmcpack.py SYMBOLIC)
-install(FILES gpaw4qmcpack.py DESTINATION bin)
```

Comment:
```
file()
install()
```
These two lines should stay

Reply:
Okay, wasn't sure and took a wrong guess. It's amended now.